module.json

```diff
@@ -1,10 +1,11 @@
 {
 	"name": "lootsheetnpc5e",
 	"title": "Loot Sheet NPC 5e",
-	"description": "This module adds an additional NPC sheet which can be used for loot containers such as chests or shopkeepers.",
-	"version": "3.4.1",
+	"description": "Add NPC sheet for looting and trading. Now with automatic npc token population options.",
```

Comment:
I feel that the old description was more informative.

Reply:
Okay, will change it back.